samples/bluetooth/broadcast_audio_source/src/main.c

```diff
@@ -34,6 +34,15 @@ BUILD_ASSERT(CONFIG_BT_ISO_TX_BUF_COUNT >= TOTAL_BUF_NEEDED,
 
 static struct bt_bap_lc3_preset preset_16_2_1 = BT_BAP_LC3_BROADCAST_PRESET_16_2_1(
 	BT_AUDIO_LOCATION_FRONT_LEFT, BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED);
+static struct bt_bap_lc3_preset preset_24_2_1 = BT_BAP_LC3_BROADCAST_PRESET_24_2_1(
+	BT_AUDIO_LOCATION_FRONT_LEFT, BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED);
+
+#if defined(CONFIG_BT_24KHZ_FREQUENCY)
+static struct bt_bap_lc3_preset *preset_active = &preset_24_2_1;
+#else
+static struct bt_bap_lc3_preset *preset_active = &preset_16_2_1;
+#endif
```

Comment:
Due to this, a warning will be thrown of preset_X_2_1 defined but not used, depending on whether 24kHz is enabled or not. If this is unacceptable, I will re-write the code so that this doesn't happen.

Reply:
You can just move the declaration of the preset in here as well to fix the warning :)